admin_manual/configuration_server/occ_command.rst

```diff
@@ -341,6 +341,21 @@ The ``config`` commands are used to configure the Nextcloud server::
   config:system:get      Get a system config value
   config:system:set      Set a system config value
 
+
+While setting a configuration value, multiple options are available:
+
+     - ``--value=VALUE`` change the configuration value
+     - ``--type=TYPE`` change the type of the value. Use carefully; can break your instance
+     - ``--lazy|--no-lazy`` set value as `lazy`
+     - ``--sensitive|--no-sensitive`` set value as `sensitive`
+     - ``--update-only`` only updates if a value is already stored
+
+.. note::
+	See `Appconfig Concepts`_ to learn more about `typed value`, `lazy` and `sensitive` flag.
+
+.. _Appconfig Concepts: https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/config/appconfig.html#concept-overview
```

Comment:
This should be a relative link.

Reply:
All link are made the same way, cannot find an example follow.
Should I just replace `https://docs.nextcloud.com/server[...]` by `/server[...]`

Reply:
I didn't realize that it's a link from admin docs to dev docs. You can leave it :+1: